eq-author/src/components/QCodeContext/index.js

```diff
@@ -1,7 +1,105 @@
-import { createContext, useContext } from "react";
+import React, { createContext, useContext, useMemo } from "react";
+import PropTypes from "prop-types";
+import CustomPropTypes from "custom-prop-types";
+
+import { RADIO } from "constants/answer-types";
 
 export const QCodeContext = createContext();
 
+export const getAllPages = ({ sections }) =>
+  sections?.flatMap(({ folders }) => folders?.flatMap(({ pages }) => pages));
```

Comment:
```suggestion
import { RADIO } from "constants/answer-types";

import { getPages } from "utils/questionnaireUtils";

export const QCodeContext = createContext();
```

I just remembered there's a function in the utils folder for this.